.env.example

```diff
@@ -1,17 +1,25 @@
-# bancho.py supports both INET and UNIX sockets
-# INET: provide an IP for SERVER_ADDR, and a port for SERVER_PORT
-# UNIX: provide a filesystem path for SERVER_ADDR, and no SERVER_PORT
-SERVER_ADDR=/tmp/bancho.sock
-SERVER_PORT=
+# for nginx reverse proxy to work through the containers, the bancho
+# server must expose itself on port 80 to be accessed on http://bancho.
+SERVER_ADDR=0.0.0.0
+SERVER_PORT=80
 
-DB_DSN=mysql://username:password@localhost:3306/database
-REDIS_DSN=redis://localhost
+HOST_PORT=443
 
-OSU_API_KEY=
+# XXX: change these values!
+MYSQL_USER=
+MYSQL_PASSWORD=
+MYSQL_DATABASE=
+# then copy the above values (user, password, database) to this:
+DB_DSN=mysql://user:password@mysql/database
```

Comment:
i think i'd rather construct the dsn in the code rather than include it here as well - i'm fine storing user/pass/dbname in env

should do the same for redis too though

Reply:
+1 on that

Reply:
completely agree